### PR TITLE
Preload fonts in Head

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Preload fonts in Head-element.
Set HTML lang.

## Justify why they are needed

Copied from "onboarding" app.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #488** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/488" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/488?utm_source=stack-comment).

File: apps/store/src/pages/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Document, { Head, Html, Main, NextScript } from 'next/document'
+import { getCDNFonts } from 'ui'
+import { getLocale } from '@/lib/l10n/locales'
+
+export default class MyDocument extends Document {
```
Comment (alebedev) [inline apps/store/src/pages/_document.tsx:5]: Does it have to be a class instead of function component?

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/pages/_document.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/pages/_document.tsx:5]: Yep, that's a next.js constraint

**Pull request merged**